### ISSUE TITLE
iOS and Android button issues when using voiceover.

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#316]
When using voiceover on iOS, all the buttons are identified two times. I believe it might be happening because of the way they are labelled. 

For instance, Notifications button should be labelled only "Notifications" as it identifies button by itself.

**Comment by zhuelle** [comment]
@payeisa need prioritation.

**Comment by payeisa** [comment]
@zhuelle Priority added.

**Comment by Rajat-Sen** [comment]
Validated!
